### ISSUE TITLE
Per position plots have a tail that is only supported by very few long reads

### ISSUE THREAD

**Issue by rhpvorderman** [rhpvorderman/sequali#215]
There should be some way to indicate this to the user. Maybe some overlay of the length distribution plot?

**Comment by SamMcGreig** [comment]
I agree that this would be a useful feature - I am looking at amplicon data and have found myself flicking between the two plots to understand how many reads I have that are longer than the expected amplicon size. 

Would it ever be useful/possible to include a plot that is anchored to the end of the reads too? Similar to how it is done in NanoQC (https://github.com/wdecoster/nanoQC) (https://gigabaseorgigabyte.wordpress.com/2017/06/15/per-base-sequence-content-and-quality-end-of-reads/). I'm not sure how much value there is in this though!

**Comment by rhpvorderman** [comment]
@s-j-mc Thank you for the suggestion. Anchoring at the end of the read is very useful. I would need to rewrite sequali a bit to be aware of the relative position from the end. Currently all counts are in a large array that is relative from the start. I do keep per position counts though, and only aggregate at the end. 

Given variable read lengths of Nanopore that is not ideal. Especially not for the adapter plot. I will make that a separate issue. 

I do have an idea how to tackle the rest of the per position plots. Rather than using the relative amounts to the total, I can use the absolute amount of bases. The number should be the largest at the beginning and then slowly degrade until the end. In combination with the end-anchored plots as you suggest this can be very informative.
